pandas/tests/indexes/datetimes/test_datetime.py

```diff
@@ -447,6 +447,40 @@ def f():
             t - offset
         pytest.raises(OverflowError, f)
 
+    def test_datetimeindex_sub_timestamp_overflow(self):
+        dtimax = pd.to_datetime(['now', pd.Timestamp.max])
```

Comment:
add the issue for these

Reply:
There is no issue for this; I noticed it when tracking down the TimedeltaIndex bug.

Reply:
sure there is, the PR number!